resources/js/components/layouts/LayoutUser/components/TheNotification.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="nav-link dropdown notification">
+    <a
+      id="dropdownMenuButton"
+      class="d-flex align-items-center"
+      data-toggle="dropdown"
+    >
+      {{ $t("route.notifications") }}
+      <span class="badge badge-pill badge-info ml-2">{{ unReadNotification }}</span>
+    </a>
+    <div
+      class="dropdown-menu dropdown-menu-right p-0 notification__list"
+      aria-labelledby="dropdownMenuButton"
+    >
+      <div v-if="notifications.length">
+        <a
+          v-for="notification in notifications"
+          :key="notification.id"
+          class="dropdown-item d-flex align-items-center justify-content-between notification__item"
+          :class="{ 'bg-info': !notification.read }"
+        >
+          {{ notification.message }}
+          <span
+            v-if="!notification.read"
+            class="btn btn-dark btn-sm ml-2"
+            @click="markAsRead(notification.id)"
+          ><small>{{ $t("notification.markAsRead") }}</small></span>
+        </a>
+      </div>
+      <div v-else>
+        <p class="dropdown-item m-0">
+          {{ $t("notification.empty") }}
+        </p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import { getNotification, markAsRead } from '@/api/notification';
+
+export default {
+  name: 'TheNotification',
+  data() {
+    return {
+      notifications: [],
+    };
+  },
+  computed: {
+    ...mapState({
+      auth: 'user',
+    }),
+    unReadNotification() {
+      return this.notifications.filter(notification => notification.read === false).length;
```

Comment:
so sánh với false luôn à chị. 
`this.notifications.filter(notification => !notification.read).length`
thế này dc rồi chứ.